example/app-histogram/src/main/java/com/mastercard/test/flow/example/app/histogram/HistogramImp.java

```diff
@@ -28,7 +28,7 @@ public Map<String, Integer> histogram( String text ) {
 	@Override
 	public Map<String, Integer> histogram( String text, String characters ) {
 		LOG.info( "Counting [{}] characters in '{}'", characters, text );
-		Optional<Set<Character>> queried = Optional.ofNullable( characters )
+		Optional<Set<Character>> queried = Optional.ofNullable( characters.toLowerCase() )
```

Comment:
No sense trying to count upper-case characters any more.
Canny reviewers (and mutation testing) will spot that this change is not exercised by our testing - we don't have any flows where the set of characters to count includes upper-case values.